astropy/io/misc/tests/test_parquet.py

```diff
@@ -145,9 +146,7 @@ def test_read_pathlikeobj(tmp_path):
     t1.add_column(Column(name="a", data=[1, 2, 3]))
     t1.write(test_file)
 
-    import pathlib
-
-    p = pathlib.Path(test_file)
+    p = Path(test_file)
```

Comment:
Just noticed this is a actually a no-op (test_file is already a Path). Most likely this test was incorrectly edited when pytest's tmp_path was introduced. I'll get back to this after lunch.